gateway-ha/src/main/java/io/trino/gateway/ha/config/MonitorConfiguration.java

```diff
@@ -42,6 +43,8 @@ public class MonitorConfiguration
 
     private Map<String, Float> metricMaximumValues = ImmutableMap.of();
 
+    private int metricRefreshSeconds = DEFAULT_METRIC_REFRESH_SECONDS;
```

Comment:
This isn't really the refresh period of the _metrics_, it's the refresh period of the _list of backends_. Maybe a name more like `backendMetricsListRefreshPeriod`? I don't love that name so open to suggestions but I think we need to make it a little more clear